docs/source/howto/run_codes.rst

```diff
@@ -346,6 +347,28 @@ At the end, you receive a confirmation, with the *PK* and the *UUID* of your new
 
     Note: remove the ``--`` prefix and replace dashes (``-``) within the keys with an underscore ( ``_`` ).
 
+
+.. note::
+
+    It is possible to run codes that are provided by a `Conda environment <https://docs.conda.io/en/latest/>`_.
+    The code configuration YAML would look something like the following:
+
+    .. code-block:: yaml
+
+        filepath_executable: 'executable-name'  # This can simply be the executable name, no need for the absolute path
+        prepend_text: conda activate environment-name
+
+    Note that the configuration is not complete but only shows the relevant lines.
+    In the `prepend_text`, the required Conda environment is activated as one normally would.
+    This allows to simply specify the target executable by its name, and there is no need to specify an absolute path.
+
+    The final requirement is that the computer should be configured to create a submission script that uses a login shell.
+    To do so, make sure the computer sets ``#!/bin/bash -l`` for the `shebang` property.
+    The ``-l`` flag in bash enforces the script to be executed using a login shell.
+    Without this, a normal non-interactive shell is used instead, which will cause the ``conda activate`` command to fail.
```

Comment:
Could also be dropped if you accept the suggestion above
```suggestion
```

If you like, you could mention the alternative of providing the absolute path and skipping `conda activate` (I guess conda packages are typically reasonable with using RPATH etc. to pick up the right dependencies, but I don't have much experience with it)